src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/ResearchSetup.java

```diff
@@ -176,6 +176,7 @@ public static void setupResearches() {
         register("hologram_projector", 166, "Holograms", 36, SlimefunItems.HOLOGRAM_PROJECTOR);
         register("capacitors", 167, "Tier 1 Capacitors", 25, SlimefunItems.SMALL_CAPACITOR, SlimefunItems.MEDIUM_CAPACITOR, SlimefunItems.BIG_CAPACITOR);
         register("high_tier_capacitors", 168, "Tier 2 Capacitors", 32, SlimefunItems.LARGE_CAPACITOR, SlimefunItems.CARBONADO_EDGED_CAPACITOR);
+        register("even_higher_tier_capacitors", 266, "Tier 3 Capacitors", 40, SlimefunItems.ENERGIZED_CAPACITOR);
```

Comment:
Please follow the existing order of ascending ids, otherwise someone who adds a new research at the bottom will miss this ONE single misplaced research inbetween with a much higher id.

Reply:
If I were in you I'd've put this to the last line and give it the next id. So no other researches' ids would be changed and anyone who adds a research later will know what's taken and what is not.

Reply:
That's what I meant, this should go at the bottom so it's back in order.

Reply:
Will work on it